PFSM Write does not sanitise offset and length

`pfsm write`, when given an offset and length of `"", ""`, accepts it, silently converts them to integers, and panics later in the program's execution, presumably when attempting to use one of them as the counter in a for loop.

Steps to reproduce:
1. Call `pfsm write <pfs_dir> <path> "" ""`.
2. Watch as the program panics.

Sample execution:

```
[terry@terry-laptop pnetserver]$ echo "foo" | pfsm write /tmp/pfs foo.txt "" ""
panic: runtime error: index out of range

goroutine 1 [running]:
main.main()
    /home/terry/code/gocode/src/github.com/cpssd/paranoid/pfsm/main.go:15 +0x145b

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
    /usr/lib/go/src/runtime/asm_amd64.s:1696 +0x1
[terry@terry-laptop pnetserver]$
```

--- a/pfsm/main.go
+++ b/pfsm/main.go
@@ -12,10 +12,12 @@ func main() {
 	var onlyArgs []string
 	var onlyFlags []string
 	for i := 0; i < len(args); i++ {
-		if args[i][0] == '-' {
-			onlyFlags = append(onlyFlags, args[i])
-		} else {
-			onlyArgs = append(onlyArgs, args[i])
+		if len(args[i]) > 0 {
+			if args[i][0] == '-' {
+				onlyFlags = append(onlyFlags, args[i])
+			} else {
+				onlyArgs = append(onlyArgs, args[i])
+			}
 		}
 	}
 	commands.ProcessFlags(onlyFlags)

